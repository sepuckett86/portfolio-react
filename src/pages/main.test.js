import React from 'react';
import { shallow } from 'enzyme';
import Main from './main';

describe('Main page', () => {
  it('renders', () => {
    const wrapper = shallow(<Main />);
    expect(wrapper).toMatchSnapshot();
  });
});
