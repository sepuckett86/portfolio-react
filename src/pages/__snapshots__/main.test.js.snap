// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main page renders 1`] = `
<Fragment>
  <IntroContainer />
  <Portfolio />
  <ContactForm />
  <GoodiesContainer />
</Fragment>
`;
